docs/service-mesh/03-03-istio-rbac.md

```diff
@@ -3,6 +3,8 @@ title: Istio RBAC configuration
 type: Details
 ---
 
+//TODO: Decide what to do with this part of docu
```

Comment:
As we will most probably support the old way of doing RBAC until Istio 1.6 I would recommend to leave this section as it is (because it is still true) and provide a note or warning that it is deprecated and the support for that will be discontinued in further releases. The note should also include a link to the Istio documentation for AuthorizationPolicy and a statement that we recommend using the new API.